<!--
  Launch file to run multi_robot_slave not on each kobuki. Multi robot slave subscribes to the 
  `\my_global_goals` topic and reads the identifier to verify if it corresponds to  a  command 
  for the kobuki on which this node runs. In addition this node calls "clear_costmaps" service 
  to avoid bad local planner behaviour
 -->
<launch>
  <arg name="robot_name" default="kobuki"/>
  <arg name="kobuki_id" default="1"/>
  <!-- Multi master slave -->
  <group ns="$(arg robot_name)">
    <node pkg="kobuki_multi_robot" type="multi_robot_slave" name="multi_robot_slave" output="screen" >
      <param name="kobuki_id" type="int" value="$(arg kobuki_id)" />
    </node>
  </group>
</launch>